### PR TITLE
small fixes for tutorial

### PR DESCRIPTION
**Pull request by opotowsky**

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -250,13 +250,14 @@ metric, you could write the following.  Call the new metric ``MaterialsSquared``
...
-    schema = [('SimId', cym.UUID), ('ResourceId', ts.INT),
-              ('NucId', cym.INT),  ('MassSquared', ts.DOUBLE)]
+    schema = [('SimId', cym.UUID), ('ResourceId', cym.INT),
+              ('NucId', cym.INT),  ('MassSquared', cym.DOUBLE)]
 
     @cym.metric(name='MaterialsSquared', depends=deps, schema=schema)
     def mats_sqrd(series):
         mats = series[0]
         rtn = mats**2
+		rtn.name = 'MaterialsSquared'
```
Comment (scopatz) [inline docs/tutorial.rst]: The indentation here seems wacky. Are you using tabs?

Comment (opotowsky) [inline docs/tutorial.rst]: aw, man, yes I need to fix that

**Comment by scopatz** [comment]
let me know when it is good to go.

**Comment by opotowsky** [comment]
now it is @scopatz!

**Comment by scopatz** [comment]
khul beings. thanks!

**Pull request merged**